portability-core/src/main/java/org/dataportabilityproject/PortabilityCoreModule.java

```diff
@@ -54,11 +60,19 @@ protected void configure() {
     install(new MicrosoftModule());
     install(new RememberTheMilkModule());
     install(new SmugmugModule());
+
+    if (commonSettings.getCloud() == SupportedCloud.GOOGLE) {
```

Comment:
Can we get away with installing this in a different way. We're bypassing the guicified CommonSettings to find out if we have a specific cloud value.

Reply:
Done, this isn't great so let me know if you can think of a better way

Reply:
Can GoogleCloudModule declare a RequireBinding for the CommonSettings?

Reply:
good idea, done!